src/components/NFTClaim/MessageBlock.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <div
+    class="flex flex-col items-start gap-[12px] relative pl-[24px] pr-[16px] py-[13px] border-[1px] border-shade-gray shadow-md rounded-[20px] rounded-bl-[8px] bg-white"
+  >
+    <div class="flex justify-start items-end gap-[18px] mt-[-22px]">
+      <Identity :avatar-url="avatarUrl" :avatar-size="40" />
+      <Label
+        preset="h5"
+        :text="
+          $t('nft_claim_claimed_message_title', { name: creatorDisplayName })
+        "
+      />
+    </div>
+    <p class="text-black font-200">{{ message }}</p>
```

Comment:
Should apply `whitespace-pre-wrap` class for rendering linebreaks

Reply:
[💄 Add white space to MessageBlock](https://github.com/likecoin/liker-land/pull/1588/commits/4ad2ac4c7205dcc94ed99c455acc76aacc210e70)